package.json

```diff
@@ -7,6 +7,10 @@
     "build": "vue-cli-service build"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.28",
```

Comment:
We're using Material Icons - they're imported with Vuetify. Please update to remove the fontawesome packages and use the Material Icons. Thanks!

Reply:
I couldn't find the exact icon with external link--this one is close.